src/components/EloGraph.js

```diff
@@ -136,9 +136,9 @@ function EloGraph(props) {
             }
         }
     }]
-    return <div className='eloGraph'>
+    return (
```

Comment:
I've seen this change in a bunch of places - why?

Reply:
Going forward will use a composite return